api/v1alpha1/contentsourcebinding_types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ContentSourceReference contains info to locate a Kind ContentSource object
```

Comment:
Wonder if we should club this with ClassReference (https://github.com/vmware-tanzu/vm-operator-api/blob/master/api/v1alpha1/virtualmachineclassbinding_types.go#L11) into a common ObjectReference.

Didn't venture into it in this PR since that would be a breaking api change.

Reply:
The k8s docs recommend using a focused class such as yours instead of a using ObjectReference https://godoc.org/k8s.io/api/core/v1#ObjectReference so keeping it as is seems okay

> Instead of using this type, create a locally provided and used type that is well-focused on your reference. For example, ServiceReferences for admission registration: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533 . +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object

Reply:
I see. Thanks for the information, Janitha.

Reply:
Yes. Using an `ObjectReference` is not preferred.